core/internal/stream/stream.go

```diff
@@ -116,6 +118,17 @@ func streamLogger(
 			Sentry: sentryClient,
 		},
 	)
+
+	// Log when we receive a shutdown signal
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, syscall.SIGINT)
+	signal.Notify(c, syscall.SIGTERM)
+	go func() {
+		sig := <-c
+		logger.Info("received shutdown signal", "signal", sig)
```

Comment:
Use `slog` directly in `main.go` instead. We install a default handler that writes to `debug-core.log`.